optimizely/notification_center.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2017, Optimizely
+# Copyright 2017-2019, Optimizely
```

Comment:
Has it really been 2 years since we touched this?

Reply:
We touched it some time last year briefly, but this was not updated.